docs/source/index.rst

```diff
@@ -98,7 +98,7 @@ repository <http://github.com/ipython/ipython>`_.
 .. seealso::
 
    `Jupyter documentation <http://jupyter.readthedocs.io/en/latest/>`__
-     The Notebook code and many other pieces formerly in IPython are now parts
+     The Notebook code and other pieces formerly in IPython, are now part
```

Comment:
I think this either needs no commas or two (the other one being after 'code'), not one.